packages/python/accern_data/accern_data.py

```diff
@@ -826,6 +845,14 @@ def iterate_range(
         start_date_only_dt = pd.to_datetime(start_date_only)
         end_date_only_dt = pd.to_datetime(end_date_only)
 
+        def get_by_date_param(date: str) -> Dict[str, str]:
+            if by_date == "published_at":
+                return {"date": date}
+            if by_date == "harvested_at":
+                return {"harvested_at": date}
```

Comment:
We're using harvested_at for pagination atm. How does this work? Does pagination take precedence?

Reply:
pagination uses `harvested_after`

Reply:
the workaround notebook used this approach